core/loadpoint.go

```diff
@@ -123,6 +123,12 @@ type LoadPoint struct {
 	chargeTimer api.ChargeTimer
 	chargeRater api.ChargeRater
 
+	// wakeUp
+	activeTimer         *ActiveTimer
+	activeDuration      time.Duration // Active current duration for wakeup calls
```

Comment:
zählt die nicht der Timer? ich hab das Gefühl das sind noch zuviele Variablen.

Reply:
hab ich reduziert und alle in den Timer geschoben.